Apps/Sandcastle/gallery/Voxels.html

```diff
@@ -309,7 +243,9 @@
           fragmentShaderText: `void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)
   {
       material.diffuse = fsInput.metadata.color.rgb;
-      material.alpha = 1.0;
+      float transparency = 1.0 - fsInput.metadata.color.a;
+      float thickness = fsInput.voxel.travelDistance * 16.0;
+      material.alpha = 1.0 - pow(transparency, thickness);
```

Comment:
```suggestion

      // To mimic light scattering, use exponential decay
      float thickness = fsInput.voxel.travelDistance * 16.0;
      material.alpha = 1.0 - pow(transparency, thickness);
```

Reply:
This is a useful formula for voxel transparency, so it would be nice to highlight it with an explanation.